Controllers/PostController.cs

```diff
@@ -27,7 +28,59 @@ public async Task<ActionResult> CreatePost([FromBody] PostDto post)
                 var result = await postService.Create(post.GetPersistentObject());
                 return Created(result.Id.ToString(), result);
             }
-            return BadRequest(post.Errors);
+            return BadRequest(post.Error);
+        }
+
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Post>> GetPost(Guid id)
+        {
+            var result = await postService.Get(id);
+            if(result == null)
+            {
+                return NotFound();
+            }
+            return result;
+        }
+
+        [HttpGet("page/{number}")]
+        public async Task<ActionResult<List<Post>>> GetPaginate(int number)
+        {
+            var queryOrder = Request.Query["order"].ToString();
+            if(!String.IsNullOrEmpty(queryOrder))
+            {
+                queryOrder = queryOrder.ToLower();
+                if(queryOrder == "desc")
+                {
+                    return await postService.GetPage(number, 10 ,true);
```

Comment:
What if our client plans to display 15 records per page?

Reply:
then addition query parameter can be passed